### PR TITLE
✨Enable conversion tracking for GTAG.

### PR DESCRIPTION
**Pull request by zikas**

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors/gtag.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
         'dr': 'DOCUMENT_REFERRER',
         'dl': 'SOURCE_URL',
       },
+      'conversion': {
+        'gclsrc': 'QUERY_PARAM(gclsrc)',
+        'hasGcl': 'IF(QUERY_PARAM(gclid, 1, 0)',
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors/gtag.js]: This seems to be missing a closing paren `)`. Same below.

Comment (calebcordry) [reply extensions/amp-analytics/0.1/vendors/gtag.js]: also `$IF`

Comment (zikas) [reply extensions/amp-analytics/0.1/vendors/gtag.js]: wow, my conflict resolution miserably failed

File: extensions/amp-analytics/0.1/vendors/gtag.js [hunk 2]
```diff
@@ -22,6 +22,12 @@ export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
         'dr': 'DOCUMENT_REFERRER',
         'dl': 'SOURCE_URL',
       },
+      'conversion': {
+        'gclsrc': 'QUERY_PARAM(gclsrc)',
+        'hasGcl': '$IF(QUERY_PARAM(gclid, 1, 0))',
```
Comment (calebcordry) [inline extensions/amp-analytics/0.1/vendors/gtag.js]: Looks like the parenthesis is still in the wrong place :(

Comment (zikas) [reply extensions/amp-analytics/0.1/vendors/gtag.js]: ok, it's time to get some coffee

**Comment by calebcordry** [comment]
LGTM assuming you manually tested.

**Pull request merged**